src/org/labkey/test/components/DomainDesignerPage.java

```diff
@@ -72,7 +116,12 @@ DomainFormPanel domainFormPanel(String domainName)
     {
         static public Locator.XPathLocator domainDesignerButton(String text)
         {
-            return Locator.tagWithClass("button", "domain-designer-button").withText(text);
+            return Locator.tagWithClass("button", "btn-success-default").withText(text);
         }
+
+        static public Locator infoAlertLoc = Locator.tagWithClass("div", "alert-info");
+        static public Locator successAlertLoc = Locator.tagWithClass("div", "alert-success");
+        static public Locator dangerAlertLoc = Locator.tagWithClass("div", "alert-danger");
+        static public Locator warningAlertLoc = Locator.tagWithClass("div", "alert-warning");
```

Comment:
I've noticed these CSS classes in Sample Management as well. We should probably put these Locators somewhere shared so we don't keep redefining them. I'm thinking a new class, similar to the existing `Locators` class: `org.labkey.test.util.BootstrapLocators`

Reply:
Okay, I'll take these and put them in a new BootstrapLocators.java class, sibling to Locators.java